src/porepy/numerics/ad/operators.py

```diff
@@ -879,6 +894,7 @@ def _find_subtree_variables(self) -> Sequence[Variable]:
                     # Effectively, this node is one step from the leaf
                     var_list.append(var)
                 elif isinstance(var, list):
+                    assert False
```

Comment:
Is this on purpose? If so, please provide a short documentation and consider deleting the code below.

Reply:
I forgot to remove it when was debugging. Deleting it.